pkg/metrics/store/counter.go

```diff
@@ -39,6 +42,14 @@ func NewStoreCounter(resManager manager.ReadOnlyResourceManager, metrics metrics
 
 func (s *storeCounter) Start(stop <-chan struct{}) error {
 	ticker := time.NewTicker(1 * time.Minute)
+	return s.start(stop, ticker)
+}
+
+func (s *storeCounter) StartWithTicker(stop <-chan struct{}, ticker *time.Ticker) error {
+	return s.start(stop, ticker)
+}
+
+func (s *storeCounter) start(stop <-chan struct{}, ticker *time.Ticker) error {
```

Comment:
I think we can rename this to `StartWithTicker` and call it from `Start`

Reply:
Done